docs/competency-api/condition.md

```diff
@@ -6,4 +6,46 @@ parent: Competency API
 has_children: true
 has_toc: false
 ---
-# Condition
\ No newline at end of file
+# Condition
+## Update Condition
+
+### Definition
+```HTTP
+https://api.competency-constructor.clark.center/competencies/:competencyId/condition
+```
+
+## Headers
+```JSON
+{
+    Bearer: Bearer.Example.bearer.token
```

Comment:
The header should be authorization but the format of the value is "Bearer eyasdffasd.asdfasdffasdf.asdfasdfasdf"

Reply:
I'm still in the boat that we should just have $BEARER_TOKEN in documentation. The actual format of the token can be in the authorization documentation since this will be across the board for any JWT.